tests/common/util.rs

```diff
@@ -337,22 +538,35 @@ impl CmdResult {
     }
 
     /// asserts that the command resulted in stderr stream output that equals the
-    /// passed in value, when both are trimmed of trailing whitespace
+    /// passed in value, when `stderr` is trimmed of trailing whitespace
+    /// `stderr_only` is a better choice unless stdout may or will be non-empty
+    #[track_caller]
+    pub fn stderr_trimmed_is<T: AsRef<str>>(&self, msg: T) -> &Self {
+        assert_eq!(self.stderr_str().trim(), msg.as_ref());
+        self
+    }
```

Comment:
Do we still need this now that we check the newlines explicitly? I think the problem you found with `tr` show that checking the newlines explicitly is the way to go.

Reply:
> Do we still need this now that we check the newlines explicitly?
Nope. I'll remove the method.